docs/en/models/yolov8.md

```diff
@@ -43,6 +43,7 @@ Each variant of the YOLOv8 series is optimized for its respective task, ensuring
 | YOLOv8-pose | `yolov8n-pose.pt` `yolov8s-pose.pt` `yolov8m-pose.pt` `yolov8l-pose.pt` `yolov8x-pose.pt` `yolov8x-pose-p6.pt` | [Pose/Keypoints](../tasks/pose.md)           | ✅        | ✅         | ✅       | ✅     |
 | YOLOv8-obb  | `yolov8n-obb.pt` `yolov8s-obb.pt` `yolov8m-obb.pt` `yolov8l-obb.pt` `yolov8x-obb.pt`                           | [Oriented Detection](../tasks/obb.md)        | ✅        | ✅         | ✅       | ✅     |
 | YOLOv8-cls  | `yolov8n-cls.pt` `yolov8s-cls.pt` `yolov8m-cls.pt` `yolov8l-cls.pt` `yolov8x-cls.pt`                           | [Classification](../tasks/classify.md)       | ✅        | ✅         | ✅       | ✅     |
+| YOLOv8-oiv7 | `yolov8n-oiv7.pt` `yolov8s-oiv7.pt` `yolov8m-oiv7.pt` `yolov8l-oiv7.pt` `yolov8x-oiv7.pt`                      | [Detection](../tasks/detect.md)              | ✅        | ✅         | ✅       | ✅     |
```

Comment:
I think it could make sense to add a column to this table for "dataset" and link to the appropriate dataset page. Not a hard requirement, just my thinking.

Reply:
Agree!